vllm/worker/hpu_model_runner.py

```diff
@@ -466,6 +466,31 @@ def _reset_rotary_cos_sin(self):
         if hasattr(self._rotary_embed_module, "sin"):
             delattr(self._rotary_embed_module, "sin")
 
+    def _set_attn_bias_chunked(self, seq_len, context_len, query_len, device,
+                               dtype):
```

Comment:
suggest to add some comments on what's the output attn_bias like based on seq_len, ctx_len and query_len. e.g. use an example.